Configuration/AppUserStore.cs

```diff
@@ -23,26 +24,19 @@ public Task AddToRoleAsync(Employee user, string roleName, CancellationToken can
         {
             DepartmentId departmentId = (DepartmentId)Enum.Parse(typeof(DepartmentId), roleName, true);
             user.Department = departmentId;
-            _context.Employees.Update(user);
```

Comment:
Why was this method removed? Wouldn't it prevent the user from updating in the DB?